### ISSUE TITLE
Extension publishing : Registration of new accounts has been disabled.

### ISSUE THREAD

**Issue by pmahend1** [mono/monodevelop#9652]
I am developing XML formatter add in  for Visual Studio for Mac.
I am in the process of bundling and publishing it. But the site is not letting me register. I tried to link my google account.

>  NOTICE
> 
> Registration of new accounts has been disabled.

Is there an alternative ?

**Comment by JoacimWall** [comment]
Hi @pmahend1  did you get any response i have the same issue.

//Joacim

**Comment by pmahend1** [comment]
@JoacimWall No. It looks like repo is inactive although Visual Studio for Mac is pretty much active. I just released the extension privately on my repo itself. I don't know who to contact now.

**Comment by hagay3** [comment]
the issue still exist, someone is on it?

**Comment by Zidanela** [comment]
I cannot register a new account either :(. Any update?

**Comment by JoacimWall** [comment]
Hello Visual studio for Mac Team any response?
Is there any info regarding Visual studio 2022 will the current extension still work?
Will Visual studio 2022 solve this issue?

//Regards Joacim

**Comment by mrward** [comment]
There is documentation about how to release extensions for Visual Studio for Mac - https://docs.microsoft.com/en-gb/visualstudio/mac/migrate-extensions

**Comment by JoacimWall** [comment]
> There is documentation about how to release extensions for Visual Studio for Mac - https://docs.microsoft.com/en-gb/visualstudio/mac/migrate-extensions

Hi I have followed that but nothing get release and no response from the email that is referens in the instructions.
I think this is only for user that has a account on the old mono dev portal(Where you signs upp for release extension, but this site has bin closed for more then a year now). So there has not bin a solution for publish new extensions for new users for more than a year now that is not a good solution.

//regards Joacim
